pandas/tests/io/excel/test_xlrd.py

```diff
@@ -39,3 +39,12 @@ def test_excel_table_sheet_by_index(datapath, read_ext):
     with pd.ExcelFile(path) as excel:
         with pytest.raises(xlrd.XLRDError):
             pd.read_excel(excel, "asdf")
+
+
+def test_read_excel_with_default(datapath):
+    path = datapath("io", "data", "test1.xls")
+    with tm.assert_produces_warning(
+        FutureWarning, check_stacklevel=False, raise_on_extra_warnings=False
+    ):
+        with pd.ExcelFile(path) as excel:
+            pd.read_excel(excel, "Sheet1")
```

Comment:
Can you separate the call of pd.ExcelFile from pd.read_excel into two different things here? They both should warn, though it's not clear from the test where the warning gets thrown

Reply:
Done - 2 tests now.